1.8/usage/tutorials/task-labels.md

```diff
@@ -0,0 +1,55 @@
+---
+post_title: Labeling Tasks
+menu_order: 12
+---
+
+This tutorial illustrates how labels can be defined using the DC/OS web interface and the Marathon HTTP API, and how information pertaining to applications that are running can be queried based on label value criteria.
+
+When you deploy applications or containers in a DC/OS cluster, you can associate a tag or label with your deployed components in order to track and report usage of the cluster by those components. For example, you may want to assign a cost center identifier or a customer number to a Mesos application and produce a summary report at the end of the month with usage metrics such as the amount of CPU and memory allocated to the applications by cost center or customer.
+
+## Assigning Labels to Applications and Tasks
+
+You can attach labels to tasks either via the **Services** tab of the DC/OS web interface or from the DC/OS CLI using the Marathon HTTP API.
```

Comment:
I think we can delete `using the Marathon HTTP API` since this is an implementation detail at this point.